hrms/payroll/doctype/payroll_period/payroll_period.py

```diff
@@ -122,8 +122,6 @@ def get_period_factor(
 
 	if relieving_date and getdate(relieving_date) < getdate(period_end):
 		period_end = relieving_date
-		if month_diff(period_end, start_date) > 1:
```

Comment:
Here the `start_date` is the start date of the salary slip. Not the payroll period/posting date. Unable to replicate this. 
In your example it says 
month_diff( 15 Dec 2024 , 1 Jan 2024 ) = 12 months
start date won't be 1st Jan 2024. It would be 1st Dec (start date of the salary slip). What are your salary slip dates?
Posting date only affects accounting. All calculation happens based on start and end dates in salary slip

Reply:
My case is creating the 1st salary slip of the year on 1 Jan 2024, so,
- start_date = 1 Jan
- period_end = 15 Dec
And so it gives 12 months and making CTC value incorrect (CTC = 10,000 * 25 = 250,000)

Reply:
@ruchamahabal any doubt? 🙏

Reply:
Hello again, is it possible to fix this bug?

Reply:
@ruchamahabal anything I can help to progress this bug?

Reply:
replicated